hack/after-build/update-generated-conversions.sh

```diff
@@ -44,14 +44,19 @@ EOF
 // AUTO-GENERATED FUNCTIONS END HERE
 EOF
 
-	mv "$TMPFILE" "pkg/api/${version}/conversion_generated.go"
+	env GOPATH=$(godep path):$GOPATH goimports -w "$TMPFILE"
```

Comment:
Why are you setting GOPATH at all?  Shouldn't this be taken care of by `kube::golang::setup_env()`  I certainly don't want random stuff I have outside of the tree somehow messing with these commands.